apps/cloud_composer/ComposerTool.cmake

```diff
@@ -19,11 +19,10 @@ function(define_composer_tool TOOL_NAME TOOL_SOURCES TOOL_HEADERS DEPS)
   add_definitions(-DQT_NO_DEBUG)
   add_definitions(-DQT_SHARED)
 
-  add_dependencies(${TOOL_TARGET} pcl_cc_tool_interface ${DEPS})
```

Comment:
Why is this removed?

Reply:
There are no more targets than in `target_link_libraries`, so it is already done automatically by CMake. A explicit call to `add_dependencies` is only necessary if you have dependencies which CMake can't see.